open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java

```diff
@@ -0,0 +1,246 @@
+package io.opensphere.core.control.keybinding;
+
+import io.opensphere.core.util.AwesomeIconSolid;
+import javafx.geometry.Insets;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+
+/**
+ * Creates a table/legend of the current Opensphere shortcut keys which pertain
+ * to General and Map controls.
+ */
+public class ControlUI extends GridPane
+{
+    /**
+     * The directory of the "plus" icon.
+     */
+    private String myPlusDir = "images/keys/plus.png";
+
+    /**
+     * The directory of the "mouse left button clicked" icon.
+     */
+    private String mouseLeft = "images/keys/MouseLeft.png";
+
+    /**
+     * The directory of the "mouse middle button clicked" icon.
+     */
+    private String mouseMiddle = "images/keys/MouseMiddle.png";
+
+    /**
+     * The directory of the "mouse right button clicked" icon.
+     */
+    private String mouseRight = "images/keys/MouseRight.png";
+
+    /**
+     * @param width the horizontal size desired.
+     * @param height the vertical size desired.
+     */
+    public ControlUI(int width, int height)
+    {
+        setMinSize(width, height);
+        setStyle("-fx-background-color : derive(-fx-base, 18%)");
+        setHgap(10);
+        setVgap(10);
+        setPadding(new Insets(5));
+
+        add(createGeneral(), 0, 0);
+        add(createGeneralMap(), 1, 0);
+        add(createMapZoom(), 0, 1);
+        add(createMapTools(), 1, 1);
+    }
+
+    /**
+     * Creates a sub-window containing Map Tool Controls.
+     * @return mainBox the sub-window contained in a GridPane.
```

Comment:
Leave an empty line between description and return, no need to give the name of the return variable

Reply:
updated